uw-opsendpoints-dotnet.sln

```diff
@@ -35,16 +35,16 @@ Global
 		{ECF49813-2B0B-441F-838F-35581A197D63}.Release|x86.Build.0 = Release|x86
 		{AB862E4C-C877-4EBB-83E9-81B60BC9AEE9}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{AB862E4C-C877-4EBB-83E9-81B60BC9AEE9}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{AB862E4C-C877-4EBB-83E9-81B60BC9AEE9}.Debug|x64.ActiveCfg = Debug|x64
-		{AB862E4C-C877-4EBB-83E9-81B60BC9AEE9}.Debug|x64.Build.0 = Debug|x64
-		{AB862E4C-C877-4EBB-83E9-81B60BC9AEE9}.Debug|x86.ActiveCfg = Debug|x86
-		{AB862E4C-C877-4EBB-83E9-81B60BC9AEE9}.Debug|x86.Build.0 = Debug|x86
+		{AB862E4C-C877-4EBB-83E9-81B60BC9AEE9}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{AB862E4C-C877-4EBB-83E9-81B60BC9AEE9}.Debug|x64.Build.0 = Debug|Any CPU
+		{AB862E4C-C877-4EBB-83E9-81B60BC9AEE9}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{AB862E4C-C877-4EBB-83E9-81B60BC9AEE9}.Debug|x86.Build.0 = Debug|Any CPU
```

Comment:
why do we flatten the build configurations to AnyCPU? I anticipate problems when used in [self contained deployments](https://docs.microsoft.com/en-us/dotnet/articles/core/preview3/deploying/).